components/brave_vpn/browser/connection/win/brave_vpn_helper/BUILD.gn

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//build/toolchain/win/midl.gni")
+import("//build/util/process_version.gni")
+import("//chrome/process_version_rc_template.gni")
+
+# Must be in a config because of how GN orders flags (otherwise /W4 will appear
+# after this, and turn it back on).
+config("no-missing-braces") {
+  # Macros invoked by WRL CoCreatableClass omit optional braces.
+  cflags = [ "-Wno-missing-braces" ]
+}
+
+source_set("common") {
+  sources = [ "brave_vpn_helper_constants.h" ]
+  deps = [ "//build:buildflag_header_h" ]
+}
+
+source_set("lib") {
+  sources = [
+    "brave_vpn_dns_delegate.h",
+    "vpn_dns_handler.cc",
+    "vpn_dns_handler.h",
+    "vpn_utils.cc",
+    "vpn_utils.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave//components/brave_vpn/browser/connection/win:utils",
+    "//brave//components/brave_vpn/common",
+    "//third_party/abseil-cpp:absl",
+  ]
+
+  libs = [
+    "Fwpuclnt.lib",
+    "Iphlpapi.lib",
+  ]
+}
+
+executable("brave_vpn_helper") {
+  sources = [
+    "brave_vpn_helper_crash_reporter_client.cc",
+    "brave_vpn_helper_crash_reporter_client.h",
+    "main.cc",
+    "service_main.cc",
+    "service_main.h",
+  ]
+
+  deps = [
+    ":lib",
+    ":version_resources",
+    "//base",
+    "//brave/components/brave_vpn/browser/connection/win/brave_vpn_helper:common",
+    "//brave/components/brave_vpn/common",
+    "//build/win:default_exe_manifest",
+    "//chrome/install_static:install_static_util",
+    "//components/browser_watcher:browser_watcher_client",
+    "//components/crash/core/app",
+    "//components/crash/core/app:crash_export_thunks",
+    "//components/crash/core/app:run_as_crashpad_handler",
+    "//components/version_info:channel",
+  ]
+  libs = [
+    "userenv.lib",  # For GetUserProfileDirectoryW()
+  ]
+}
+
+process_version_rc_template("version_resources") {
+  sources = [ "brave_vpn_helper.ver" ]
+  output = "$target_gen_dir/brave_vpn_helper_exe.rc"
+}
+
+source_set("unit_tests") {
+  testonly = true
+  assert(is_win)
```

Comment:
I think this assert should be in global in this file.

Reply:
browser/ui/webui/settings/brave_import_bulk_data_handler_unittest.cc